### ISSUE TITLE
Allowing an editable number of multiple choice answers for questions

### ISSUE THREAD

**Issue by franzmoro** [cul-2016/quiz#291]
To allow for survey questions of the following type:
- strongly disagree
- disagree
- neutral
- agree
- strongly agree

Not providing the "neutral" option can leave this to 4.

**Comment by iteles** [comment]
Are we hard-coding in the options? Surely they need to be editable?

**Comment by franzmoro** [comment]
@iteles they are editable, but although the options can have any text, our db structure currently support only up to 4 options (a, b, c, d)

**Comment by stianr** [comment]
Very good point. Some multiple choice quizzes also use 5 options, which we couldn't handle at the moment. In the previous version we allowed between 2 and 4 options. Trade off between useful flexibility and combination of time to update interface and db and formatting/presentation issues if we allow loads of options. How long would it take to allow 5 or more? It would definitely be desirable if it's not too much time.

**Comment by sohilpandya** [comment]
We've had a chat about implementing this feature, and here is what we think should happen:

- instead of providing users with 4 or 5 options, it would probably be better to provide `n` number of options so that we future proof it. (allowing lecturers to have generic number of options per question) 
- Implementing this would mean a restructure of the database as well as the UI for the front end
- This would probably require **3 days** to implement and test. 

What are your thoughts with regards to this?

**Comment by stianr** [comment]
Agree - allowing n options is ideal. Hard to judge whether it's worth the three days without knowing what kind of other functionality we would be losing, given time available.

I'd ideally like to take stock at the end of the first sprint to decide what to prioritise. But that would depend on whether delaying a decision would impact on other aspects of the project, and make it a larger job if we decided to implement it towards the end of the project.

**Comment by iteles** [comment]
@stianr Agreed. I'm back in town on Monday and will book in some time to sit with the team and take stock of the overall status of the project as well as booking in our end of sprint demo. We'll discuss this issue specifically and update.

**Comment by iteles** [comment]
We didn't have time to carry this out in the end but leaving in the backlog as a good feature for future iterations!
